test/pylib/scylla_cluster.py

```diff
@@ -125,6 +125,8 @@ def make_scylla_conf(mode: str, workdir: pathlib.Path, host_addr: str, seed_addr
 
         'reader_concurrency_semaphore_serialize_limit_multiplier': 0,
         'reader_concurrency_semaphore_kill_limit_multiplier': 0,
+        'view_update_reader_concurrency_semaphore_serialize_limit_multiplier': 0,
+        'view_update_reader_concurrency_semaphore_kill_limit_multiplier': 0,
```

Comment:
I added these to follow the behavior of the regular read concurrency semaphore, but I'm still not sure why we're using no limits here for tests. @denesb do you remember why it was set this way?

Reply:
Setting these limits to 0 effectively disables the semaphore's anti OOM protection.
Tests usually run with small amount of memory and we don't want the semaphore to start killing reads in a random test.